### PR TITLE
Update modulemanager.datalayer.json

### PR DESCRIPTION
**Pull request by rubenalanis**

### CODE REVIEW

File: dataLayer/modulemanager.datalayer.json [hunk 1]
```diff
@@ -1,2 +1,17 @@
-{
-}
+MMdataLayer = [{
```
Comment (BigDataKatha) [inline dataLayer/modulemanager.datalayer.json:1]: Please change the single quotation marks to double quotation marks which is the standard we are using for all other data layers.

File: dataLayer/modulemanager.datalayer.json [hunk 2]
```diff
@@ -1,2 +1,17 @@
...
+    'languageIsoCode': 'en',
+    'countryIsoCode ': 'US',
+    'originAirportIataCodes ': 'MIA,TXL,SXF',
```
Comment (BigDataKatha) [inline dataLayer/modulemanager.datalayer.json:4]: This should be an array. You can check the airTRFX proposed data layer of how arrays are supposed to look like

File: dataLayer/modulemanager.datalayer.json [hunk 3]
```diff
@@ -1,2 +1,17 @@
...
+    'destinationAirportIataCodes': 'LHR,MIL,LAX',
+    'fareClass': 'ECONOMY',
+    'journeyType': 'ONE_WAY',
+    'isLoggedIn': 'TRUE',
```
Comment (BigDataKatha) [inline dataLayer/modulemanager.datalayer.json:8]: Certain fields should be grouped under loyalty Program.

**Review by BigDataKatha — CHANGES_REQUESTED** [review]
Hi Ruben, I left some comments for changes, but please check the whole data layer to apply the comments

File: dataLayer/modulemanager.datalayer.json (hunk 1 above)
Comment (rubenalanis) [reply dataLayer/modulemanager.datalayer.json:1]: Hi Kath - The standard dataLayer scripts in the analytics world use single quotation marks: https://developers.google.com/tag-manager/devguide. Should we keep using them for future dataLayers?

File: dataLayer/modulemanager.datalayer.json (hunk 3 above)
Comment (rubenalanis) [reply dataLayer/modulemanager.datalayer.json:8]: This will be added

File: dataLayer/modulemanager.datalayer.json (hunk 2 above)
Comment (rubenalanis) [reply dataLayer/modulemanager.datalayer.json:4]: I noticed that in the proposed dataLayer this is the format for the variable:
"destinationAirportIataCodes": [
            "BER", "TXL", "SXF"
        ],
Is there a reason why this is like this? Are we going to separate all of them by array?

The format could also be:
'destinationAirportIataCodes': 'LHR,MIL,LAX'
To include all airports in a single value. 

Let me know what you think.

File: dataLayer/modulemanager.datalayer.json (hunk 1 above)
Comment (BigDataKatha) [reply dataLayer/modulemanager.datalayer.json:1]: As discussed, let's use double quotation marks here.

File: dataLayer/modulemanager.datalayer.json (hunk 2 above)
Comment (BigDataKatha) [reply dataLayer/modulemanager.datalayer.json:4]: I added a comment in the Chat yesterday, but the array follows JSON best practices.

File: dataLayer/modulemanager.datalayer.json (hunk 3 above)
Comment (BigDataKatha) [inline dataLayer/modulemanager.datalayer.json:8]: Note that for booleans, we just populate true or false in lowercase without quotation marks following again JSON best practices.

**Pull request closed without merging**